### PR TITLE
Add section about text array handling in full text indexes

### PR DESCRIPTION
**Pull request by zimmre**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 1]
```diff
@@ -175,6 +177,25 @@ class FulltextIndexTest extends DocumentingTest {
       p("A complete description of the Lucene query syntax can be found in the link:https://lucene.apache.org/core/8_2_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#package.description[Lucene documentation].")
     }
 
+    section( "Handling of Text Array properties", id="administration-indexes-fulltext-search-query") {
+      p(
+        """If indexes property contains text array each element of this array will be analysed independently and all produced terms will be associated with the same property name.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]:
```suggestion
        """If the indexed property contains a text array, each element of this array is analyzed independently and all produced terms are associated with the same property name.
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 2]
```diff
@@ -175,6 +177,25 @@ class FulltextIndexTest extends DocumentingTest {
...
+           #This mean that when quering such index node or relationship will match if any of array elements match the query.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]:
```suggestion
           #This means that when querying such an indexed node or relationship, there is a match if any of the array elements match the query.
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 3]
```diff
@@ -175,6 +177,25 @@ class FulltextIndexTest extends DocumentingTest {
...
+           #For example both following queries match the same node while referring different elements""".stripMargin('#'))
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala]:
```suggestion
           #For example, both of the following queries match the same node while referring different elements""".stripMargin('#'))
```

**Review by AlexicaWright — COMMENTED** [review]
Some editorial comments.

**Comment by AlexicaWright** [comment]
Also, the tests are failing.

**Comment by zimmre** [comment]
@AlexicaWright thank you! test should be fixed now

**Review by AlexicaWright — APPROVED** [review]
👍 Great! Thank you @zimmre , feel free to merge!

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 4]
```diff
@@ -175,6 +178,25 @@ class FulltextIndexTest extends DocumentingTest {
       p("A complete description of the Lucene query syntax can be found in the link:https://lucene.apache.org/core/8_2_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#package.description[Lucene documentation].")
     }
 
+    section( "Handling of Text Array properties", id="administration-indexes-fulltext-search-query") {
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala:181]:
```suggestion
section( "Handling of Text Array properties", id="administration-indexes-fulltext-search-text-array-properties") {
```

Comment (zimmre) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala:181]: luckily I have another change in this file and fixed this section id too https://github.com/neo4j/neo4j-documentation/pull/1569

**Pull request merged**